recipe/activate-cross-r-base.sh

```diff
@@ -0,0 +1,6 @@
+if [[ "$CONDA_BUILD_STATE" != "test" && "$build_platform" != "$target_platform" ]]; then
```

Comment:
This should error out when not used inside of `conda-build` or use the same approach as in https://github.com/conda-forge/r_clang_activation-feedstock/blob/master/recipe/activate-r-clang_win-64.bat where a temporary `R_MAKEVARS_SITE` is created.